### ISSUE TITLE
ACS 7.2 - Active Sessions

### ISSUE THREAD

**Issue by aborroy** [OrderOfTheBee/ootbee-support-tools#192]
When accessing to Active Sessions option via Alfresco Admin Console

http://localhost:8080/alfresco/s/ootbee/admin/active-sessions

404 error is returned with a popup error window.

The error from alfresco log is the following one:

```
Caused by: freemarker.core.UnexpectedTypeException: For "?c" left-hand operand: Expected a number or boolean, but this evaluated to an extended_hash+string (org.mozilla.javascript.Undefined wrapped into f.e.b.StringModel):
zz-alfresco-1            | ==> connectionPoolData.maxActive  [in template "org/orderofthebee/support-tools/admin/ootbee-support-tools/active-sessions.get.html.ftl" at line 34, column 26]
zz-alfresco-1            |
zz-alfresco-1            | The failing instruction (print stack trace for 3 more):
zz-alfresco-1            | ==> ${connectionPoolData.maxActive?c}  [in template "org/orderofthebee/support-tools/admin/ootbee-support-tools/active-sessions.get.html.ftl" at line 34, column 24]
```

**Comment by aborroy** [comment]
The problem seems related with the upgrade of Apache Commons DBCP, that is used to get database metrics in
https://github.com/OrderOfTheBee/ootbee-support-tools/blob/master/repository/src/main/resources/alfresco/templates/webscripts/org/orderofthebee/support-tools/admin/ootbee-support-tools/active-sessions.lib.js#L28

Some of the methods/properties has been changed or deprecated:

```
ctxt = Packages.org.springframework.web.context.ContextLoader.getCurrentWebApplicationContext();
dataSource = ctxt.getBean('defaultDataSource', Packages.javax.sql.DataSource);

logger.log(dataSource.initialSize);
logger.log(dataSource.numActive);
logger.log(dataSource.maxActive);
logger.log(dataSource.minIdle);
logger.log(dataSource.numIdle);
logger.log(dataSource.maxIdle);
logger.log(dataSource.maxWait);
logger.log(dataSource.url);
logger.log(dataSource.driverClassName);

---

10
2
undefined
10
8
10
undefined
jdbc:postgresql://postgres:5432/alfresco
org.postgresql.Driver
```

**Comment by AFaust** [comment]
Addressed in #190

**Comment by AFaust** [comment]
PR #190 has been merged, so this issue is also resolved.
